config/settings.rb

```diff
@@ -3,9 +3,11 @@
 require_relative '../app/services/foreman/env_settings_loader'
 
 root = File.expand_path(File.dirname(__FILE__) + "/..")
-settings_file = Rails.env.test? ? 'config/settings.yaml.test' : 'config/settings.yaml'
+settings_files = Rails.env.test? ? ['config/settings.yaml.test', 'config/settings.yaml.test.local'] : ['config/settings.yaml', 'config/settings.yaml.local']
```

Comment:
I'm not in favor of this since it complicates the logic. We already have many layers of config files that are loaded. In `config/boot_settings.rb` to get the Rails version:
* `config/settings.yaml.dist`
* `config/settings.yaml`

Then in this file:
* `config/settings.yaml` or `config/settings.yaml.test`
* Any environment variables

Note that we already have a few to set the logging level.